master/internal/config/config.go

```diff
@@ -461,6 +507,51 @@ type TLSConfig struct {
 	Key  string `json:"key"`
 }
 
+// Validate implements the check.Validatable interface for the TokenConfig.
+func (t *TokenConfig) Validate() []error {
+	var errs []error
+	if t.MaxLifespanDays < 1 && t.MaxLifespanDays != InfiniteTokenLifespan {
+		errs = append(
+			errs,
+			errors.New("max token lifespan must be greater than 0 days, unless set to -1 for"+
+				"infinite lifespan"),
+		)
+	}
+	if t.DefaultLifespanDays < 1 && t.DefaultLifespanDays != InfiniteTokenLifespan {
```

Comment:
Allow having `t.MaxLifespanDays` and `t.DefaultLifespanDays` able to set to 0, this will give the Cluster Admin the ability to shut the token feature